pom.xml

```diff
@@ -27,6 +29,18 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>${logback.version}</version>
```

Comment:
Better to add full title and without dot `.`.

Like: `logback-core-version`

Reply:
fix